### ISSUE TITLE
Suggestion for Reducing Shellcode Size

### ISSUE THREAD

**Issue by hlldz** [TheWover/donut#68]
Hi,

Donut created a ~35KB shellcode for a simple .EXE using the MessageBox API. Do you have any advice to reduce shellcode size with Donut?

Sample Code:
```
#include <Windows.h>

int main() {
	MessageBoxA(NULL, "A", "B", MB_OK);
	return 0;
}
```

**Comment by TheWover** [comment]
Hi, @hlldz . Have you tried using the compression options available in v0.9.3? 

-z | engine | Pack/Compress the input file. 1=None, 2=aPLib, 3=LZNT1,  4=Xpress, 5=Xpress Huffman. Currently, the last three are only supported  on Windows.
-- | -- | --

**Comment by hlldz** [comment]
Hi TheWover,

I was using version 0.9.2. Upgrading to version 0.9.3 and yes, the size has been greatly reduced using the feature you mentioned.

Thank you.
